docs/modules/ROOT/pages/contributors/secret-rotation.adoc

```diff
@@ -0,0 +1,15 @@
+= Secret Rotation
+
+This document lists every secret KubeArchive uses in its processes that requires
+rotation.
+
+== KubeArchive Organization
+
+link:https://github.com/organizations/kubearchive/settings/secrets/actions[Organization Secrets]
+
+* OCI_PASSWORD: password for the robot account named after the variable OCI_USERNAME.
+Go to the
+link:https://quay.io/organization/kubearchive?tab=robots[KubeArchive's Quay Organization]
+and regenerate the token for the appropiate robot.
+* `KUBEARCHIVE_BOT_SECRET`: to be done.
+* `KUBEARCHIVE_RENOVATE_PRIVATE_KEY`: to be done.
```

Comment:
This is done by going to https://github.com/organizations/kubearchive/settings/apps/kubearchive-renovate and clicking the button `generate a new client secret`